symbology/render-multilayer-symbols/src/main/java/com/esri/samples/render_multilayer_symbols/RenderMultilayerSymbolsSample.java

```diff
@@ -0,0 +1,459 @@
+    var multilayerPointSymbol = new MultilayerPointSymbol(List.of(vectorMarkerSymbolLayer));
+    
+    // create point graphic using the symbol and add it to the graphics overlay
+    var graphic = new Graphic(new Point(-150, 20 - offset, SpatialReferences.getWgs84()), multilayerPointSymbol);
+    graphicsOverlay.getGraphics().add(graphic);
+    
+  }
+
+  /**
+   * Adds new graphics constructed from multilayer polyline symbols.
+   *
+   * @param dashSpacing The pattern of dots/dashes used by the line.
+   * @param offset The value used to keep a consistent distance between symbols in the same column.
+   */
+  private void addLineGraphicsWithMarkerSymbols(List<Double> dashSpacing, double offset) {
+
+    // create a dash effect from the provided values
+    var dashGeometricEffect = new DashGeometricEffect(dashSpacing);
+
+    // stroke used by line symbols
+    var solidStrokeSymbolLayer = new SolidStrokeSymbolLayer(3.0, ColorUtil.colorToArgb(Color.RED), List.of(dashGeometricEffect));
+    solidStrokeSymbolLayer.setCapStyle(StrokeSymbolLayer.CapStyle.ROUND);
+
+    // create a polyline for the multilayer polyline symbol
+    var polylineBuilder = new PolylineBuilder(SpatialReferences.getWgs84());
+    polylineBuilder.addPoint(new Point(-30, 20-offset));
+    polylineBuilder.addPoint(new Point(30, 20-offset));
+
+    // create a multilayer polyline symbol from the solidStrokeSymbolLayer
+    var multilayerPolylineSymbol = new MultilayerPolylineSymbol(List.of(solidStrokeSymbolLayer));
+
+    // create a polyline graphic with geometry using the symbol created above, and add it to the graphics overlay
+    graphicsOverlay.getGraphics().add(new Graphic(polylineBuilder.toGeometry(), multilayerPolylineSymbol));
+  }
+
+  /**
+   * Adds new graphics constructed from multilayer polygon symbols.
+   *
+   * @param angles A list containing the angle at which to draw any fill lines within the polygon.
+   * @param offset The value used to keep a consistent distance between symbols in the same column.
+   */
+  private void addPolygonGraphicsWithMarkerSymbols(List<Double> angles, double offset) {
+    var polygonBuilder = new PolygonBuilder(SpatialReferences.getWgs84());
+    polygonBuilder.addPoint(new Point(60, 25-offset));
+    polygonBuilder.addPoint(new Point(70, 25-offset));
+    polygonBuilder.addPoint(new Point(70, 20-offset));
+    polygonBuilder.addPoint(new Point(60, 20-offset));
+
+    // create a stroke symbol layer to be used by patterns
+    SolidStrokeSymbolLayer strokeForHatches = new SolidStrokeSymbolLayer(2, ColorUtil.colorToArgb(Color.RED),
+      List.of(new DashGeometricEffect()));
+
+    // create a stroke symbol layer to be used as an outline for aforementioned patterns
+    SolidStrokeSymbolLayer strokeForOutline = new SolidStrokeSymbolLayer(1, ColorUtil.colorToArgb(Color.BLACK),
+      List.of(new DashGeometricEffect()));
+
+    var symbolLayerArray = new SymbolLayer[angles.size() + 1];
```

Comment:
very neat! could we add a brief comment to describe what is happening here for the user?